tools/lint-docstring

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+
+import argparse
+import importlib
+import sys
+from os import getcwd, listdir
+from os.path import isfile, join
+from typing import Dict, List, Tuple, cast
+
+
+developer_doc = join(getcwd(), 'docs', 'developer-file-overview.md')
+
+
+def get_docstrings() -> Tuple[Dict[str, str], Dict[str, List[str]]]:
+    root = join(getcwd(), 'zulipterminal')
+    folder_paths = {
+        'zulipterminal': root,
+        'zulipterminal/cli': join(root, 'cli'),
+        'zulipterminal/config': join(root, 'config'),
+        'zulipterminal/ui_tools': join(root, 'ui_tools')
+    }
```

Comment:
This does work, but we should be able to at minimum automate this dictionary generation, and at most we can likely look inside all of `zulipterminal` with a `glob`?

I'd also expect the `zulipterminal` to be able to be a parameter we pass in, as long as we scan through sub-directories automatically without needing this dict.